sdk/translation/Azure.AI.Translation.Document/tests/Infrastructure/DocumentTranslationSamplesConstructor.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Azure.AI.Translation.Document.Tests;
+using Azure.Core.TestFramework;
+
+namespace Azure.AI.Translation.Document.Samples
+{
+    public partial class DocumentTranslationSamples : DocumentTranslationLiveTestBase
+    {
+        public DocumentTranslationSamples(bool async)
+            : base(async, RecordedTestMode.Live)
```

Comment:
we shouldn't need to set the `RecordedTestMode.Live` as the CI will set it for us.
You should be able to only do `: base(async)`